osu.Game/Online/Chat/DrawableLinkCompiler.cs

```diff
@@ -52,6 +77,41 @@ private void load(OsuColour colours)
 
         protected override IEnumerable<Drawable> EffectTargets => Parts;
 
+        public MenuItem[] ContextMenuItems
+        {
+            get
+            {
+                List<MenuItem> items = new List<MenuItem>();
+                string text;
+
+                switch (link.Action)
```

Comment:
I would check all links as this is a very general class. I got this when right clicking the mapper name on a beatmap card:

![osu!_WMqHk6BKRM](https://user-images.githubusercontent.com/35318437/235334934-271416fd-68a2-4d50-a4cc-5f283d8d3937.png)

![rider64_gvcJqUptqR](https://user-images.githubusercontent.com/35318437/235334814-87cbfeb5-7d6d-4fd7-a828-27699d5ff2f5.png)